openfisca_us/parameters/gov/states/mo/tax/income/rates.yaml

```diff
@@ -0,0 +1,68 @@
+description: The tax rates for individual residents in Missouri. Thresholds are <=.
+brackets:
+- rate:
```

Comment:
I think you're hitting the error because this (and below) needs to be indented

Reply:
got it, thanks